api/DedicatedWorkerGlobalScope.json

```diff
@@ -0,0 +1,332 @@
+{
+  "api": {
+    "DedicatedWorkerGlobalScope": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "3.5"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": "10.6"
+          },
+          "opera_android": {
+            "version_added": "11.5"
+          },
+          "safari": {
+            "version_added": "4"
+          },
+          "safari_ios": {
+            "version_added": "5.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "ononline_onoffline": {
```

Comment:
Looks like this is actually on WorkerGlobalScope:
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/ononline
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onoffline

Let's remove it from this table.

Reply:
👍